### PR TITLE
fix gcc9 compilation error

### PR DESCRIPTION
**Pull request by dimitry-**

### CODE REVIEW

File: src/OrbitCaptureClient/CaptureClient.cpp [hunk 1]
```diff
@@ -58,6 +58,7 @@ static CaptureOptions::InstrumentedFunction::FunctionType InstrumentedFunctionTy
         FunctionInfo_OrbitType_FunctionInfo_OrbitType_INT_MAX_SENTINEL_DO_NOT_USE_:
       UNREACHABLE();
   }
+  UNREACHABLE();
```
Comment (dpallotti) [inline src/OrbitCaptureClient/CaptureClient.cpp:61]: Why not just restore the `default` in the switch? I don't think we should use those values directly.

Comment (dimitry-) [reply src/OrbitCaptureClient/CaptureClient.cpp:61]: because we want a compile time error when enum class changes.

Comment (dpallotti) [reply src/OrbitCaptureClient/CaptureClient.cpp:61]: But with the addition of this `UNREACHABLE()` would you still get that compile time error now? If I get this correctly this new `UNREACHABLE()` acts like `default` now.

Comment (dimitry-) [reply src/OrbitCaptureClient/CaptureClient.cpp:61]: yes, if there is no default in the switch - and you add another value to enum you get compile time error. (it is a warning promoted to a error)

Comment (dpallotti) [reply src/OrbitCaptureClient/CaptureClient.cpp:61]: Makes sense.

**Review by dpallotti — APPROVED** [review]

**Review by florian-kuebler — APPROVED** [review]
LGTM, thanks

**Pull request merged**